### DISCUSSION TITLE
What type of deviation in heights in a bed can the automatic levelling handle?

### DISCUSSION THREAD

**Discussion by marcosgeorgopoulos** [mriscoc/Ender3V2S1#498] [Q&A]
Hi,

What type of deviation in heights in a bed can the automatic levelling handle?

I've been struggling for over a week trying to get a single thing to print, I'm currently running your latest ABL firmware.

I'm getting this mesh which is almost identical across two different sheets of G10

[bed meshes](https://imgur.com/a/uTmRzPm)

- brought the bed and nozzle up to print temp.
- I levelled the corners using the wizard.
- created a 5x5 mesh
- set my z-offset.

My g-code start code for cura  has been taken directly from this site and I can see the mesh is being activated. ( little line under 'z' on screen? )

I have tried adjusting the z-offset by small single increments, but it is still printing either two close in some parts ( to the point in which the filament cannot come out ) or too high the the point that its not sticking to the bed.

[Here is an example of what the first layer should look like](https://imgur.com/a/bcATYFR)

[And here is the rubbish I'm consistently getting. ](https://imgur.com/a/F7oLkKz)

It is as if the auto-levelling is over compensating go simply cannot handle the bed I have

Any help is appreciated.

**Comment by mriscoc** [comment]
Check the eccentric wheels and rail for the Y axis.

**Reply by marcosgeorgopoulos** [reply]
Thank you. I've adjusted the eccentric wheels, they were loose.  But I am still having the same issue after tightening them. Does it sound like a bent y-axis to you?

**Reply by sgtpepperaut** [reply]
have you manually trimmed the bed already and set the proper Zprobe offset? this was my issue because i didn't RTFM.

**Reply by marcosgeorgopoulos** [reply]
yes, as per the details in the original question.

**Comment by grrmisfit** [comment]
this is a known issue with the S1, I have 2 of them and sometimes the mesh works other times it just does its own thing. currently mine likes to tell me the back right corner is to high when its clearly not. Im not sure if its a marlin bug or hardware but I have heard people moving to klipper fixes the issue. But this issue is known for stock and custom firmware

**Reply by marcosgeorgopoulos** [reply]
I started experiencing this issue after moving my printer around to take off the extruder.  So I believe it is more of a hardware issue.

My question was 'What type of deviation in heights in a bed can the automatic levelling handle?'  

From looking at Klippers documentation it appears to be 0.25 mm ( I'm not sure over what distance that is though )

**Reply by mriscoc** [reply]
I did put some coins to test leveling and the printer had no problems compensating for that deviation.
